app/decorators/cd_api/defence_counsel_decorator.rb

```diff
@@ -22,20 +22,22 @@ def name_and_status_blank?
     def name
       return nil unless first_name || middle_name || last_name
 
-      [first_name, middle_name, last_name].compact.reject(&:empty?).join(' ')
+      [first_name, middle_name, last_name].filter_map { |n| n&.capitalize }.reject(&:empty?).join(' ')
     end
 
     def formatted_name
       name || t('generic.not_available')
     end
 
     def formatted_status
-      status || t('generic.not_available').downcase
+      status || I18n.t('generic.not_available').downcase
```

Comment:
If we have to add `I18n` then it needs adding to the method above

Reply:
Not required here, sorry. will delete